server/src/DBTablesAction.cc

```diff
@@ -1114,7 +1115,7 @@ string ActionsQueryOption::Impl::makeConditionTemplate(void)
 	// host_id;
 	const ColumnDef &colDefHostId = COLUMN_DEF_ACTIONS[IDX_ACTIONS_HOST_ID];
 	cond += StringUtils::sprintf(
-	  "((%s IS NULL) OR (%s=%%" PRIu64 "))",
+	  "((%s IS NULL) OR (%s=%%s))",
```

Comment:
I think we should use the next convention.

When using LocalHostId object: localhostId.
printf("%" FMT_LOCLAL_HOST_ID, localhostId.c_str());

When wrapping DBTermCStringProvider: rhs
printf("%s", rhs(localhostId));

The latter way has the advantages that we can omit c_str()
and replace the type of 'hocalhostId' w/o other changes
because rhs() has some overload versions.
This is one of the reasons why DBTermCStringProvider has been introduced.

Now, the line you pointed out is the latter case. So I use '%s' not  FMT_LOCAL_HOST_ID.